src/build/__tests__/build-test.js

```diff
@@ -149,4 +165,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [CustomDirective] } };
+    const target = build(schemaSource + queryWithDirectiveSource, config);
+    expectSchemasEqual(target, SchemaWithDirective);
+  });
```

Comment:
Could you import `SCHEMA_CONFIG_KEY` from the main file here so that if the key changes in the future there is no need to change the test?

Would look something like:

```js
it('should allow a schema configuration to be passed via SCHEMA_CONFIG_KEY', () => {
```

This enhancement you're doing "enables" directive support but more generally it allows to configure schema via the `__schema` key, which is what we should test in `build`. If you look at the [tests](https://github.com/bloveit/graphql-utilities/blob/master/src/build/__tests__/buildSchema-test.js#L71) in `buildSchema` you'll see that directive support was already being supported in a way.